scripts/ccf/acl/up.sh

```diff
@@ -42,14 +28,10 @@ acl-up() {
     # Create a member cert
     export KMS_MEMBER_CERT_PATH="$WORKSPACE/member0_cert.pem"
     export KMS_MEMBER_PRIVK_PATH="$WORKSPACE/member0_privk.pem"
-    openssl ecparam -out "$KMS_MEMBER_PRIVK_PATH" -name "secp384r1" -genkey
-    openssl req -new -key "$KMS_MEMBER_PRIVK_PATH" -x509 -nodes -days 365 -out "$KMS_MEMBER_CERT_PATH" -"sha384" -subj=/CN="ACL Client Cert"
-
-    # Create a user cert
     export KMS_USER_CERT_PATH="$WORKSPACE/user0_cert.pem"
     export KMS_USER_PRIVK_PATH="$WORKSPACE/user0_privk.pem"
-    openssl ecparam -out "$KMS_USER_PRIVK_PATH" -name "secp384r1" -genkey
-    openssl req -new -key "$KMS_USER_PRIVK_PATH" -x509 -nodes -days 365 -out "$KMS_USER_CERT_PATH" -"sha384" -subj=/CN="ACL Client Cert"
+    ccf-member-create member0
```

Comment:
What happens if `acl-up` is called by accident (e.g. by carelessly invoking make commands)? Can we prevent overwriting the existing cert here?

Reply:
Done in https://github.com/microsoft/azure-privacy-sandbox-kms/pull/435/commits/3fa6b3ab8d36bd894a5791bb3dffe9767ed19037